release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,79 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
+
+This release introduces support for multiple Availability Zones, as well as more graceful upgrades, and improved worker labeling.
+
+### Important upgrade notes
+
+- Due to Azure limitations, upgraded clusters do not support multiple Availability Zones. In order for a cluster to be able to scale across multiple Availability Zones, it must be created using the v11.1.0 release.
+
+---
+
+### azure-operator v2.9.0
+
+#### Minor changes
+
+- Public IP of Azure LB is now re-used for ingress LB.
```

Comment:
```suggestion
- Cluster scaling is less disruptive.
```
I had to drop the Public IP re-use. There was something wrong with the configuration that broke cluster networking and I hadn't caught that during testing. Instead I figured that it might make sense to mention about less disruptive scaling that is there now.